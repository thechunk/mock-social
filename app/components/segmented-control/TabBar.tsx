import * as React from 'react';
import {PureComponent} from 'react';
import {View} from 'react-native';

export default class TabBar extends PureComponent<ITabBar> {
    render() {
        return (
            <View style={this.props.style}>
                {this.props.children}
            </View>
        );
    }
}
