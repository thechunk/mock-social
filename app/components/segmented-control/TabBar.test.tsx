import * as React from 'react';
import {shallow} from 'enzyme';
import TabBar from './TabBar';

describe('TabBar', () => {
    it('renders correctly', () => {
        expect(shallow(<TabBar />)).toMatchSnapshot();
    });
});