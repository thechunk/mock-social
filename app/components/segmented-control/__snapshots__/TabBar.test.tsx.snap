// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TabBar renders correctly 1`] = `<View />`;
